src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -877,6 +877,7 @@ public void parse(InputFile inputFile, Format ext, int type, boolean thumbOnly,
 					// Create the thumbnail image using the Thumbnailator library
 					try {
 						thumb = ImagesUtil.scaleImage(Files.readAllBytes(file.toPath()), 320, 320, ScaleType.MAX, ImageFormat.JPEG, false);
+						ffmpeg_parsing = false;
```

Comment:
@valib Another question is why thumbnail parsing with MPlayer is done "the first time" (```thumbOnly == false```) but all other thumbnail parsing is done the second time. Maybe that's what SHOULD be changed to make this more logical, but I don't know if there's a particular reason it's done this way or if it's just because of previous modifications where the author hasn't fully understood the logic.